step2-06/README.md

```diff
@@ -51,44 +53,38 @@ function reducers(state, action) {
 }
 ```
 
-In this way, it is very predictable which reducer changed the part of the state.
+With the second example, it is easy to understand which reducer changed a given part of the state.
 
-# `getState()`
+## `getState()`
 
-To examine the state of the store, you can call `getState()` to get the current snapshot of the state.
-
-```ts
-store.getState();
-```
+To examine the state of the store, you can call `store.getState()` to get a snapshot of the current state.
 
-## Discussion
+In general, you should only store serializable things in the state so that you can easily save or transfer it. You can even save this state into a browser's local storage and restore for the next boot of your application!
```

Comment:
I think this point should be called out more prominently (and maybe earlier) for a couple reasons:
- It's important to know that only serializable things should go in state
- The ability to save and restore state seems like a big deal and super magical compared to a lot of other state management methods

Reply:
I like it here because this is the first time i'm able to get a state. We're finally able to `getState` and initialize state with `createStore` ... and finally, able to change it with `reducer`.